tests/rptest/tests/schema_registry_test.py

```diff
@@ -170,6 +170,87 @@ def get_compat_dataset(type: SchemaType) -> TestCompatDataset:
     assert False, f"Unsupported schema {type=}"
 
 
+class TestNormalizeDataset(NamedTuple):
+    type: SchemaType
+    schema_base: str
+    schema_canonical: str
+    schema_normalized: str
+
+
+def get_normalize_dataset(type: SchemaType) -> TestNormalizeDataset:
+    if type == SchemaType.AVRO:
+        return TestNormalizeDataset(type=SchemaType.AVRO,
```

Comment:
just to confirm - the thing that's being tested here is effectively that normalization is _always_ a side effect of canonicalization for `SchemaType.AVRO`. is that right?

Reply:
> just to confirm - the purpose of this test case is to show that normalization is _always_ a side effect of canonicalization for `SchemaType.AVRO`. is that right?

Yes